docusaurus.config.js

```diff
@@ -72,6 +73,11 @@ const config = {
             position: 'left',
             label: 'Documentation',
           },
+          {
+            href: 'https://app.divviup.org',
+            label: 'App',
```

Comment:
maybe "Divvi Up" instead of app?

Reply:
If you think that's clearer. I was thinking `App` just to disambiguate whether that link would take you to `app.divviup.org` or `divviup.org`